### DISCUSSION TITLE
电脑控制安卓端经常崩溃   Android 7.1   armv7

### DISCUSSION THREAD

**Discussion by haisong88** [community/community#121180] [Actions]
### Select Topic Area

Bug

### Body

电脑控制安卓端经常崩溃   Android 7.1   1.2.2正常   崩溃版本1.2.4   1.2.3

**Comment by ghostinhershell** [comment]
Hi @haisong88, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
